payment-management-service/payment-management-client/src/main/resources/api.yaml

```diff
@@ -0,0 +1,165 @@
+openapi: 3.0.0
+x-samples-languages:
+  - java
+info:
+  description: Payment management service for WFIiS AGH game shop project
+  version: 1.0.0
+  title: Payment Management Service
+
+paths:
+  /:
+    get:
+      tags:
+        - Payments
+      description: Returns all Payment methods
+      operationId: getAllPaymentMethods
+      parameters:
+        - in: header
+          name: Authorization
+          schema:
+            type: string
+          required: true
+          description: Authorization token
+      responses:
+        "200":
+          description: Returns all payments
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/PaymentDto"
+
+    post:
+      tags:
+        - Payments
+      description: Create new Payment
+      operationId: createPayment
+      parameters:
+        - in: header
+          name: Authorization
+          schema:
+            type: string
+          required: true
+          description: Authorization token
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/PaymentDto"
+      responses:
+        "200":
+          description: Returns created Payment
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/PaymentDto"

```

Comment:
This endpoint will be used only by order management service, so it should not be accessible from the outside, so probably authorization token is not required

Reply:
I guess it doesn't matter and token should be present regardless. In case of potential future changes

Reply:
@theratty do you approve?

Reply:
(y)